documentation/antora.yml

```diff
@@ -25,3 +25,4 @@ asciidoc:
     link-oracle-plugin-snapshot: 'https://oss.sonatype.org/service/local/artifact/maven/redirect?r=snapshots&g=io.debezium&a=debezium-connector-oracle&v=LATEST&c=plugin&e=tar.gz'
     link-db2-plugin-snapshot: 'https://oss.sonatype.org/service/local/artifact/maven/redirect?r=snapshots&g=io.debezium&a=debezium-connector-db2&v=LATEST&c=plugin&e=tar.gz'
     link-cassandra-plugin-snapshot: 'https://oss.sonatype.org/service/local/artifact/maven/redirect?r=snapshots&g=io.debezium&a=debezium-connector-cassandra&v=LATEST&c=plugin&e=tar.gz'
+    link-kafka-docs: 'https://kafka.apache.org/documentation'
```

Comment:
In my discussion with the downstream docs team, the `antora.yml` file is not consumed downstream.  This is what has allowed us to add our own attributes here and they are able to easily override them downstream.  We most likely need to coordinate with them so they also add this attribute to their build process as well.

Reply:
Ah, I see. @bhardesty, can you take a look at this PR then, too? IIUC, you'd need to add a definition of this variable in the downstream docs build, too. Thanks!

Reply:
Summoning @fbolton, too. I'm about to merge this PR, so it'd mean you'd have to add that variable to the downstream docs build environment, too. Is that ok?

Reply:
I just added the attribute for the downstream docs, so they should be good now.